sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -409,3 +409,11 @@ export function hasUnderscoreResponse<T extends object>(
 ): result is T & HttpResponseField<any> {
   return Object.prototype.hasOwnProperty.call(result, "_response");
 }
+/**
+ * @internal
+ */
+export function formatLabelForConfigurationSetting(
+  label: string | null | undefined
+): string | undefined {
+  return label === null ? undefined : label;
```

Comment:
This is tripping me up.

Just curious.. what happens if we send "null" as is without this cast?